transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -260,29 +244,37 @@ static jobject netty_io_uring_setup(JNIEnv *env, jclass class1, jint entries, jo
         return NULL;
     }
 
-    jobject ioUringSubmissionQueue = (*env)->NewObject(
-        env, ioUringSubmissionQueueClass, ioUringSubmissionQueueMethodId,
-        (jlong)io_uring_ring.sq.khead, (jlong)io_uring_ring.sq.ktail,
+    jlong submissionArrayElements[] = {
+        (jlong)io_uring_ring.sq.khead,
+        (jlong)io_uring_ring.sq.ktail,
         (jlong)io_uring_ring.sq.kring_mask,
-        (jlong)io_uring_ring.sq.kring_entries, (jlong)io_uring_ring.sq.kflags,
-        (jlong)io_uring_ring.sq.kdropped, (jlong)io_uring_ring.sq.array,
-        (jlong)io_uring_ring.sq.sqes, (jlong)io_uring_ring.sq.ring_sz,
-        (jlong)io_uring_ring.cq.ring_ptr, (jint)ring_fd, submitCallback);
-
-    jobject ioUringCompletionQueue = (*env)->NewObject(
-        env, ioUringCompletionQueueClass, ioUringCommpletionQueueMethodId,
-        (jlong)io_uring_ring.cq.khead, (jlong)io_uring_ring.cq.ktail,
+        (jlong)io_uring_ring.sq.kring_entries,
+        (jlong)io_uring_ring.sq.kflags,
+        (jlong)io_uring_ring.sq.kdropped,
+        (jlong)io_uring_ring.sq.array,
+        (jlong)io_uring_ring.sq.sqes,
+        (jlong)io_uring_ring.sq.ring_sz,
```

Comment:
We don't need to store `ring_sz` for either CQ or SQ. It's only used for unmap and we can just calculate it there.

Reply:
I would prefer to keep it as it is so we don't need the logic for calculation in multiple places